recipe/conda_forge_ci_setup/upload_or_check_non_existence.py

```diff
@@ -92,7 +92,7 @@ def upload_or_check(recipe_dir, owner, channel, variant):
 
     # Azure's tokens are filled when in PR and not empty as for the other cis
     # In pr they will have a value like '$(secret-name)'
-    if token.startswith('$('):
+    if token and token.startswith('$('):
```

Comment:
Should it be `or`?

Reply:
Token is either `None` or a `str` so `and` is needed to prevent it doing `None.startswith('$(')`. It's equivalent to:

```python
if token is None:
    pass
elif token.startswith('$('):
    token = None
```

Reply:
Yep, you're right